### PR TITLE
fix: erc20 to erc20 ictt bridge chapter

### PR DESCRIPTION
**Pull request by owenwahlgren**
This PR cleans up the erc-20 -> erc20 bridge using ICTT.
- Fix contract paths
- Rename / fix environment variables
- Add info about AvaCloud relayer
- Add submission process for listing on Core

### CODE REVIEW

**Comment by owenwahlgren** [comment]
waiting on gform from avacloud 👍

File: content/common/avalanche-ictt/setup-manual.mdx [hunk 1]
```diff
@@ -94,15 +94,15 @@ Currency Symbol:   NAT
...
-- Funded Address (with 100 tokens),
+- Main funded account EWOQ (with 100 tokens),
 - Interchain Messaging Registry on C-chain, and
 - Interchain Messaging Registry on Avalanche L1
 
-Set these parameters as environment variables so that we can manage them easily and also use them in the commands later.
+**IMPORTANT!** Set these parameters as environment variables so that we can manage them easily and use them in commands later.
 
 ```bash
 export FUNDED_ADDRESS=<Funded Address (with 100 tokens)>
```
Comment (martineckardt) [inline content/common/avalanche-ictt/setup-manual.mdx:104]: Let's add this to the `devcontainer.json` of the Starter Kit and set it to the EWOQ Address

File: content/common/avalanche-ictt/setup-manual.mdx [hunk 2]
```diff
@@ -94,15 +94,15 @@ Currency Symbol:   NAT
...
 export TELEPORTER_REGISTRY_C_CHAIN=<Interchain Messaging Registry on C-chain>
```
Comment (martineckardt) [inline content/common/avalanche-ictt/setup-manual.mdx:105]: This one should be constant too, so we can add that as well to the devcontainer.json

File: content/common/avalanche-ictt/setup-manual.mdx [hunk 3]
```diff
@@ -94,15 +94,15 @@ Currency Symbol:   NAT
...
-export TELEPORTER_REGISTRY_SUBNET=<Interchain Messaging Registry on Avalanche L1>
+export TELEPORTER_REGISTRY_L1=<Interchain Messaging Registry on Avalanche L1>
```
Comment (martineckardt) [inline content/common/avalanche-ictt/setup-manual.mdx:106]: Not sure about this one, please try if it's always the same. If it is, lets also add to `devcontainer.json`

**Comment by martineckardt** [comment]
We have duplicate instructions to create a blockchain. The first time in the [Environment Setup](https://avalanche-academy-git-ictt-sendandcall-ava-labs.vercel.app/course/interchain-token-transfer/02-avalanche-starter-kit/03-create-blockchain) chapter using the [content/commmon/avalanche-starter-kit/create-default-blockchain instructions](https://github.com/ava-labs/avalanche-academy/blob/erc20-erc20-ictt-fix/content/common/avalanche-starter-kit/create-default-blockchain.mdx) and then later in the [ERC-20-to-ERC-20 Bridge](https://avalanche-academy-git-ictt-sendandcall-ava-labs.vercel.app/course/interchain-token-transfer/06-erc-20-to-erc-20-bridge/01-erc-20-to-erc-20-bridge) chapter using the [content/commmon/avalanche-ictt/setup-manual.mdx](https://github.com/ava-labs/avalanche-academy/pull/111/files#diff-67d81a597cae8f61b08962c0973bff9c42620d40959fa4b443085189c44c5f48) instructions.

- [ ] Can we delete the `content/commmon/avalanche-ictt/setup-manual.mdx`? I don't see a difference, the output is outdated and want to reduce as much as possible
- [ ] The blockchain created in the `Environment Setup` chapter never gets used as far as I can tell. Should we just move the chapter after the `Tokens`, `Token Bridging` and `Interchain Token Transfer` chapter? Then we can use the blockchain created in the `Environment Setup` for the bridge
- [ ] Add the environment variables that are constant to the `devcontainer.json` of the starter kit

**Review by martineckardt — CHANGES_REQUESTED** [review]

**Comment by owenwahlgren** [comment]
should be merged alongside https://github.com/ava-labs/avalanche-starter-kit/pull/33

**Review by martineckardt — APPROVED** [review]

**Pull request merged**